client/src/layers/Kanban/index.tsx

```diff
@@ -3,12 +3,33 @@ import Styled from '@/layers/Kanban/style';
 import KanbanColumn from '@/components/KanbanColumn';
 import KanbanModal from '@/components/KanbanColumn/KanbanDeleteModal';
 import { StatusType } from '@/types/story';
+import { useSocketReceive } from '@/lib/hooks';
+import { getStoryById } from '@/lib/api/story';
+import { useSetRecoilState } from 'recoil';
+import storyListAtom from '@/recoil/story';
 
 const Kanban = () => {
   const dragRef = useRef<number | null>(0);
   const dragOverRef = useRef<number | null>(0);
   const dragCategory = useRef<StatusType>('TODO');
   const dragOverCateogry = useRef<StatusType>('TODO');
+  const setStoryList = useSetRecoilState(storyListAtom);
+
+  useSocketReceive('NEW_STORY', async (storyId: number) => {
+    const data = await getStoryById(storyId);
+    if (!data) return;
+    setStoryList((prev) => [...prev, data]);
```

Comment:
immer를 사용하시면 더 명확하게 표현하실 수 있습니다! (사실 안쓰는게 더 깔끔한데 도입했다고 자랑스럽게 말하고 다녀서..ㅋㅋㅋ)

```suggestion
    setStoryList((prev) => produce(prev, (draft) => draft.push(data));
```